frontend/src/components/pages/GlobalForms/WaiversTab.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ClauseInfo } from "../../../types/AdminTypes";
+import WaiverSectionCard from "./WaiverSectionCard";
+  
+  interface WaiversTabProps {
+    clauses: Array<ClauseInfo>;
+  }
+  const WaiversTab = ({
+    clauses
+  }: WaiversTabProps): JSX.Element => {
+
+    function intToChar(number: number) {
```

Comment:
I think we can move this function to the Card component. Can we also rename this function something like `getSectionTitle` which takes an index and just returns the full title? (eg: takes index = 1 and returns "Section B")

This would make it easier to read the code for others instead of having to understand how you're converting the index to character